app/src/main/res/values/strings.xml

```diff
@@ -1,29 +1,29 @@
-    <string name="email_or_username_is_missing_please_check_login_credentials_and_try_again">Email or username is missing. Please check login credentials and try again.</string>
-    <string name="application_icon">Application icon</string>
-    <string name="jetpack_compose_logo">Jetpack Compose logo</string>
-    <string name="next">Next</string>
+    <string name="an_error_occurred" translatable = "false">An error occurred</string>
+    <string name="there_is_a_problem_with_a_network_connection_please_check_your_network_connection_and_try_again" translatable = "false">There is a problem with a network connection. Please check your network connection and try again.</string>
+    <string name="server_is_not_reachable_the_server_might_be_down_or_some_other_connection_problem_occurred" translatable = "false">Server is not reachable. The server might be down, or some other connection problem occurred.</string>
+    <string name="unexpected_error_happened_sorry" translatable = "false">Unexpected error happened. Sorry.</string>
+    <string name="users" translatable = "false">Users</string>
+    <string name="api_info" translatable = "false">API info</string>
+    <string name="logout" translatable = "false">Logout</string>
+    <string name="email" translatable = "false">E-mail</string>
+    <string name="password" translatable = "false">Password</string>
+    <string name="info" translatable = "false">Info</string>
+    <string name="welcome" translatable = "false">Welcome</string>
+    <string name="toggle_visibility" translatable = "false">Toggle visibility</string>
+    <string name="this_login_will_only_accept_email_eve_holt" translatable = "false">This login will only accept E-mail: \'eve.holt@reqres.in\'. On the other hand, any password value is valid.</string>
+    <string name="ok" translatable = "false">OK</string>
+    <string name="sign_in" translatable = "false">Sign in</string>
+    <string name="password_can_not_be_empty_please_check_entered_password_and_tra_again" translatable = "false">Password can not be empty. Please check entered password and try again.</string>
+    <string name="unexpected_storage_exception_happened_while_trying_to_log_in_please_try_again" translatable = "false">Unexpected storage error happened while trying to login. Please try again.</string>
+    <string name="entered_user_is_not_found_please_check_info_for_valid_user_info" translatable = "false">Entered user is not found. Please check info for valid user info.</string>
+    <string name="email_or_username_is_missing_please_check_login_credentials_and_try_again" translatable = "false">Email or username is missing. Please check login credentials and try again.</string>
+    <string name="application_icon" translatable = "false">Application icon</string>
+    <string name="jetpack_compose_logo" translatable = "false">Jetpack Compose logo</string>
+    <string name="next" translatable = "false">Next</string>
```

Comment:
Ovaj `translatable="false"` ide samo ako string resurs ne sadrži "raw string" tj tekst, nego referencira neki drugi string literal, to trenutno ne treba, ali će trebati ako bi išli dodati druge jezike pa će ti kompajler znati reći koji string nije preveden na tipa hrvatski i neće se htjeti kompajlirati, da ne moraš golim okom gledati